package.json

```diff
@@ -89,6 +89,7 @@
     "knex": "^0.20.15",
     "knex-paginate": "^1.2.1",
     "lodash": "^4.17.21",
+    "meow": "^8.0.0",
```

Comment:
I don't _think_ this is necessary in our own package.json, and as you noted it also doesn't fix the problem with dependencies.

Reply:
Yeah - already gone.